### PR TITLE
feat(charting): add the ability to edit chart title and axis labels to Charting Authoring PD-1726 PD-1721

### PR DESCRIPTION
**Pull request by CarlaCostea**
- add the ability to edit chart title and axis labels to Charting Authoring
- added configurable placeholders for lables and title

### CODE REVIEW

File: packages/plot/src/root.jsx [hunk 1]
```diff
@@ -155,13 +166,14 @@ export class Root extends React.Component {
               side="left"
               text={labels.left}
               disabledLabel={disabledLabels}
-              placeholder={labelsPlaceholders.left}
+              placeholder={isChart ? labelsPlaceholders?.labels : labelsPlaceholders.left}
```
Comment (andreeimiron) [inline packages/plot/src/root.jsx:169]: i suggest to revert the changes to : `placeholder={labelsPlaceholders.left}` and to keep labelsPlaceholders an object with this structure `{ left: '', right: '', top: '', bottom: '' }`

Comment (CarlaCostea) [reply packages/plot/src/root.jsx:169]: sounds good for me.
